### ISSUE TITLE
Solution because for the moment lost money ^^

### ISSUE THREAD

**Issue by dragoncity17** [ccxt/binance-trade-bot#387]
I do have purchases and resales (see screen) but I don't earn money, on the contrary I lose money on my balance...

How can I adjust it to buy and resell to make profit ?:)

![alt text](https://snipboard.io/pc0RxQ.jpg)

**Comment by belentorrente** [comment]
So far my experience with this trade bot is that if the coins are going up, you will make a profit, because it will select the coins that are increasing in price the fastest. However, if all the coins are down, you will inevitably lose money.

**Comment by dragoncity17** [comment]
Ok, and are you in profit during 1 month or more ?

**Comment by belentorrente** [comment]
No, I haven't been testing it for a month yet. The market went down a lot but I wasn't using it at the moment.
